.github/workflows/create-or-update-release-pr.yml

```diff
@@ -0,0 +1,59 @@
+name: Create or Update Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: Release type
+        options:
+          - patch
+          - minor
+      release_scope:
+        type: choice
+        description: Release Scope
+        options:
+          - experimental
+          - sdk
+          - all
+
+jobs:
+  create-or-update-release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          repository: opentelemetrybot/opentelemetry-js
+          ref: main
+          token: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
+      - name: Sync with upstream
+        run: |
+          git remote show origin
+          git remote add upstream https://github.com/open-telemetry/opentelemetry-js.git
+          git fetch upstream
+          git reset --hard upstream/main
+          git push origin main --force
+
+      - uses: actions/setup-node@v4
+        with:
+          cache: 'npm'
+          cache-dependency-path: |
+            package-lock.json
+          node-version: 22
+      - run: npm install -g npm@latest
+
+      - name: Bootstrap
+        run: npm ci
+
+      - name: Create/Update Release PR
+        run: |
+          git config user.name opentelemetrybot
+          git config user.email 107717825+opentelemetrybot@users.noreply.github.com
+          npm run github:create_or_update_release_pr
+        env:
+          GITHUB_TOKEN: ${{ secrets.OPENTELEMETRYBOT_GITHUB_TOKEN }}
+          RELEASE_SCOPE: ${{ github.event.inputs.release_scope }}
+          RELEASE_KIND: ${{ github.event.inputs.release_scope }}:${{ github.event.inputs.release_type }}
+          RELEASE_PR_REMOTE: origin
+          RELEASE_PR_OWNER: opentelemetrybot
```

Comment:
Reviewer note: If we don't have a fork on @opentelemetrybot yet and we need to release, the way to create a PR would be to set all these env vars and locally and running the `npm run github:create_or_update_release_pr` script above (`gh` CLI needs to be logged in).

For me this would be:

RELEASE_SCOPE: [[any one of these values]](https://github.com/open-telemetry/opentelemetry-js/pull/4849/files#diff-a467fd1c5e031cc87807e7a34f57f89debc18b64d84bd8c37c0ca5162013565dR16-R18)
RELEASE_KIND: $RELEASE_SCOPE: [[any one of these values]](https://github.com/open-telemetry/opentelemetry-js/pull/4849/files#diff-a467fd1c5e031cc87807e7a34f57f89debc18b64d84bd8c37c0ca5162013565dR10-R11)
RELEASE_PR_REMOTE: `origin` (the name of the remote that I usually push to)
RELEASE_PR_OWNER: `dynatrace-oss-contrib` (the organization that owns my fork of the repo)